specs/PushNotifications/ToastNotifications-spec.md

```diff
@@ -0,0 +1,1026 @@
+        NotificationNotFound, // The progress operation failed to find a toast to process updates
+    };
+
+    [experimental]
+    enum ToastPriority
+    {
+        Default, // The notification should have default behavior in terms of delivery and display priority during connected standby mode.
+        High, // The notification should be treated as high priority. For desktop PCs, this means during connected standby mode the incoming notification can turn on the screen for Surface-like devices if it doesn't have a closed lid detected.
+    };
+
+    [experimental]
+    // Represent a toast Notification construct
+    runtimeclass ToastNotification
+    {
+        // Initialize a new Toast using an XML Payload.
+        ToastNotification(Windows.Data.Xml.Dom.XmlDocument payload);
+
+        // Unique identifier used to replace a notification within a group.
+        String Tag;
+
+        // Unique identifier for a toast group in the app
+        String Group;
+
+        // A unique identifier for the Toast generated by the platform.
+        UInt32 ToastId;
+
+        // The notification Xml Payload
+        Windows.Data.Xml.Dom.XmlDocument Payload{ get; };
+
+        // Gets or sets additional information about the toast progress.
+        ToastProgressData ProgressData;
+
+        // Gets or sets the time after which a toast notification should not be displayed.
+        Windows.Foundation.DateTime ExpirationTime;
+
+        // Indicates whether the toast will remain in the Action Center after a reboot.
+        Boolean ExpiresOnReboot;
+
+        // Gets or sets the priority for a Toast.
+        // Hints on how and at what urgency level a notification should be presented to the user (whether to wake up the screen, etc).
+        ToastPriority Priority;
+
+        // Gets or sets whether a toast's pop-up UI is displayed on the user's screen.
+        Boolean SuppressDisplay;
+    };
+
+    [experimental]
+    // Supports Toast related operations for all prior displayed Toasts in Action Centre
+    runtimeclass ToastNotificationHistory
+    {
+        // Removes a specific toast with a specific toastIdentifier from Action Centre
+        Windows.Foundation.IAsyncAction RemoveWithIdentiferAsync(UInt32 toastIdentifier);
+
+        // Removes a toast having a specific tag
+        Windows.Foundation.IAsyncAction RemoveWithTagAsync(String tag);
+
+        // Removes a toast having a specific tag and group
+        Windows.Foundation.IAsyncAction RemoveWithTagGroupAsync(String tag, String group);
+
+        // Remove all toasts for a specific group
+        Windows.Foundation.IAsyncAction RemoveGroupAsync(String group);
+
+        // Removes all the toasts for the App from Action Centre
+        Windows.Foundation.IAsyncAction RemoveAllAsync();
+
+        // Gets all the toasts for the App from Action Centre
+        Windows.Foundation.IAsyncOperation<Windows.Foundation.Collections.IVector<ToastNotification> > GetAllAsync();
```

Comment:
nit, `>>` is ok

Reply:
Unfortunately MIDL doesnt like it. 
Severity	Code	Description	Project	File	Line	Suppression State
Error	MIDL2025	[msg]syntax error [context]: expecting > or , near ">>"	WindowsAppRuntime_DLL	D:\reunion\ProjectReunion\dev\ToastNotifications\ToastNotifications.idl	174